demos/openshift-install/installer.sh

```diff
@@ -174,15 +174,37 @@ while [ "$1" != "" ]; do
 done
 
 if [ "$OPENSHIFT_URL" == "" ]; then
-  echo "Missing value in --ocp-url - exiting"
-  exit 1
+  read -p 'Please specify OpenShift URL to be configured:' OPENSHIFT_URL
+  if [ "$OPENSHIFT_URL" == "" ]; then
+    echo "Missing value in OpenShift URL - exiting"
+    exit 1
+  fi
 fi
 
 if [ "$PROJECT_NAME" == "" ]; then
-  echo "Missing value in --project-name - exiting"
-  exit 1
+  read -p 'Please specify project name to be configured:' PROJECT_NAME
```

Comment:
Can you please also add a default value here?

```suggestion
  read -p 'Please specify the OpenShift project name to configured: ' PROJECT_NAME
```